contrib/kind.sh

```diff
@@ -840,7 +896,8 @@ install_ovn() {
   # leverage the kubeadm well-known label node-role.kubernetes.io/control-plane=
   # to choose the nodes where ovn master components will be placed
   for n in $MASTER_NODES; do
-    kubectl label node "$n" k8s.ovn.org/ovnkube-db=true node-role.kubernetes.io/control-plane="" --overwrite
+    kubectl label node "$n" k8s.ovn.org/ovnkube-db=true k8s.ovn.org/zone-name=global node-role.kubernetes.io/control-plane="" k8s.ovn.org/ovnkube-master="" k8s.ovn.org/ovnkube-zone-ncm="" --overwrite
```

Comment:
Having `k8s.ovn.org/zone-name` both as a label and an annotation can be confusing. If this is actually a "designated" vs "actual" kind of thing, it might be useful to convey that in the name.

Reply:
Ok.

 Let me explore on a better name.  Let me know if you've any suggestions.